insights/specs/default.py

```diff
@@ -544,6 +546,17 @@ def md5chk_file_list(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovs_vsctl_list_bridge = simple_command("/usr/bin/ovs-vsctl list bridge")
     ovs_vsctl_show = simple_command("/usr/bin/ovs-vsctl show")
+
+    @datasource(PsAuxcww, context=HostContext)
+    def java_cmd_and_pkg(broker):
+        """Command: echo the command and package string"""
+        pkg_cmd = list()
+        for cmd in _get_running_commands(broker, 'java'):
+            pkg_cmd.append("{0} {1}".format(cmd, _get_package(broker, cmd)))
+        return pkg_cmd
+
+    package_provides_java = foreach_execute(java_cmd_and_pkg, "/usr/bin/echo %s")
```

Comment:
Since you are just echoing the data, would it make more sense to use create a `DatasourceProvider` object here and return that? Or do you actually want a separate datasource for each one?

Reply:
On second thought, I think the way you implemented this spec is OK.  DatasourceProvider in the datasource is not necessary.

Reply:
@bfahr - Ack, Thanks